docs/en/plugins/README.md

```diff
@@ -0,0 +1,413 @@
+
+##### 1. Create a route
+First, let’s expose an endpoint, ‘/api/v1/discussion_tags.json’, which will route to the index action in a DiscussionTagsController:
+
+```
+# plugins/loomio_tags/plugin.rb (within the setup! method)
+plugin.use_route :get, '/discussion_tags', 'discussion_tags#index'
+```
+
+##### 2. Create a controller
+Now, we’ll define our index action in a new controller:
+
+```
+# plugins/loomio_tags/controllers/discussion_tags_controller.rb
+class ::API::DiscussionTagsController < API::RestfulController
+  def index
+    instantiate_collection { |collection| collection.where('discussions.key': params[:discussion_keys]) }
+    respond_with_collection
+  end
+
+  private
+
+  def accessible_records
+    resource_class.joins(:discussion)
+                             .where('discussions.id': Queries::VisibleDiscussions.new(user: current_user).pluck(:id))
+  end
+end
+```
+This may look a little confusing, but here’s the gist:
+- **accessible_records** defines the DiscussionTags the current user has access to, namely those associated with discussions which the user can see
+- **instantiate_collection** passes in those accessible records as a collection, and assigns the result of the block to @discussion_tags
+- **respond_with_collection** takes @discussion_tags and serializes it according to our API standards (more on that [here](https://github.com/loomio/tech-manual/blob/master/using_the_loomio_api.md#api-response-information))
+
+##### 3. Create a serializer
+Finally, we’ll define how those discussion tags we’re responding with appear in our API response. A simplified version:
+
+```
+# plugins/loomio_tags/serializers/discussion_tag_serializer.rb
+class DiscussionTagSerializer < ActiveModel::Serializer
+  embed :ids, include: true
+  attributes :id, :tag_name, :tag_color, :discussion_id
+end
+```
+
+You can check out some of the other serializers we've written in [app/serializers](https://github.com/loomio/loomio/tree/master/app/serializers)
+
+Alright, with that out of the way, we can access the tags we create through our API. Next up? Part 4 will cover how to add these tags to the actual interface. We’re getting close!
+
+### PART 4: Display created tags in the frontend
+
+Now that we’ve got some plumbing going, we can start displaying our hard work in the interface.
+
+The angular interface is comprised primarily of things called ‘components’, which is a collection of a directive, a template, and a stylesheet. We can then slot these into ‘outlets’ in the interface.
```

Comment:
The angular interface is comprised primarily of things called ‘components’~~, which is a~~**. A component consists of** a directive, a template, and a stylesheet. We can  ~~then~~ slot these **components** into ‘outlets’ in the interface. [Just a suggestion]